File: src/main/resources/CFiles/testPalindromic.c

int main() {
    char input[] = "qwertyuiopoiuytrewq";
    int len;
    scanf("%s", input);
    len = strlen(input);
    int first = 0, last = len - 1;
    int isPalindromic = 1;
    while (first < last) {
        if (input[first] == input[last]) {
            ++first;
            --last;
        } else {
            isPalindromic = 0;
            break;
        }
    }

    if (isPalindromic) {
        printf("Input text is palindromic.\n");
    } else {
        printf("Input text is not palindromic.\n");
    }
}

File: testPalindromic.js
let input = [] = "abcba";
let length_t = 5;
function main() {
let first = 0, last = length_t - 1;
let isPalindromic = 1;
while (first< last)
{
if (input[first] == input[last] )
{
++first;
--last;
}
else
{
isPalindromic=0;
break;
}
}
if (isPalindromic)
{
alert("The given text is palindromic.\n");
}
else
{
alert("The given text is not palindromic.\n");
}
}
main();
